tools/README.md

```diff
@@ -37,6 +37,14 @@ A description of the files or sub-directory is given in the table below.
 | ----------------- | ----------- |
 | [XS README](./xs/MGXS_README.md) | Description of the contents of the `xs` directory |
 
+### Angular Quadrature Plotting
 
+Files are located in `ang_quad_plotting`
+
+A description of the files or subdirectory is given in the table below.
+
+| File or Directory | Description |
+| ----------------- | ----------- |
+| `plot_quadrature.py` | Python script for plotting angular discretization (see [ANG_QUAD_PLOT README](./ang_quad_plotting/ANG_QUAD_PLOTTING_README.md)) | 
```

Comment:
this file is not plotting any quadrature, but only SLDFE quadratures, so rename the script so that its goal is more explicit